tempesta_fw/sock_srv.c

```diff
@@ -1536,7 +1536,7 @@ tfw_cfg_sg_ratio_verify(TfwSrvGroup *sg)
 	int count = 0;
 
 	if (sg->flags & (TFW_SG_F_SCHED_RATIO_DYNAMIC
-			 || TFW_SG_F_SCHED_RATIO_PREDICT))
+			 | TFW_SG_F_SCHED_RATIO_PREDICT))
```

Comment:
good catch!

Reply:
This is actually was spotted by @i-rinat once. Seems I've lost the attribution in commit message. It was at some branch